doc/sphinx/user/cookbooks/teaching-setups.md

```diff
@@ -10,90 +11,21 @@ more suitable for running them in the classroom, or they include additional
 ideas for what parameters can be changed to learn more about the physical
 behaviour that controls the model results.
 
-##### Introduction to Geophysics
+## Introduction to Geophysics
 
 *This section was contributed by Juliane Dannberg, based on the course
 &ldquo;Introduction to Geophysics&rdquo; at University of Florida.*
 
 The course is designed to teach general concepts of geophysics, and it
 includes the following cookbooks:
 
-1.  (using the files in [cookbooks/convection-box-particles/][])
-
-2.  (using the files in [cookbooks/heat_flow/][])
-
-3.  (using the files in [cookbooks/onset_of_convection/][])
-
-4.  (using the files in [cookbooks/magnetic_stripes/][])
-
-<div id="refs" class="references csl-bib-body hanging-indent">
-
-<div id="ref-FBTGS19" class="csl-entry">
-
-Fraters, M. R. T., W. Bangerth, C. Thieulot, A. C. Glerum, and W. Spakman.
-2019. &ldquo;Efficient and Practical Newton Solvers for Nonlinear Stokes
-Systems in Geodynamics Problems.&rdquo; *Geophysics Journal International* 218
-(2): 873&ndash;94. <https://doi.org/10.1093/gji/ggz183>.
-
-</div>
-
-<div id="ref-heister_aspect_methods2" class="csl-entry">
-
-Heister, Timo, Juliane Dannberg, Rene Gassm&ouml;ller, and Wolfgang Bangerth.
-2017. &ldquo;High Accuracy Mantle Convection Simulation Through Modern
-Numerical Methods. II: Realistic Models and Problems.&rdquo; *Geophysical
-Journal International* 210 (2): 833&ndash;51.
-<https://doi.org/10.1093/gji/ggx195>.
-
-</div>
-
-<div id="ref-KHB12" class="csl-entry">
-
-Kronbichler, M., T. Heister, and W. Bangerth. 2012. &ldquo;High Accuracy
-Mantle Convection Simulation Through Modern Numerical Methods.&rdquo;
-*Geophysical Journal International* 191: 12&ndash;29.
-<https://doi.org/10.1111/j.1365-246X.2012.05609.x>.
-
-</div>
-
-<div id="ref-T15" class="csl-entry">
-
-Tosi, N., C. Stein, L. Noack, C. H&uuml;ttig, P. Maierova, H. Samual, D. R.
-Davies, et al. 2015. &ldquo;A Community Benchmark for Viscoplastic Thermal
-Convection in a 2-d Square Box.&rdquo; *Geochem.&nbsp;Geophys.&nbsp;Geosyst.*
-16: 2175&ndash;96.
-
-</div>
-
-</div>
+1. {ref}`sec:cookbooks:convection-box-particles` (using the files in `cookbooks/convection-box-particles/`)
 
-[1] You can also extend <span class="smallcaps">ASPECT</span> using plugins
```

Comment:
Where did this note and the notes below go?

Reply:
The first two went into cookbooks/overview, the third went into benchmarks/index